cloud-aws-common/src/test/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsPlatformResourcesTest.java

```diff
@@ -317,6 +317,40 @@ public void collectEncryptionKeysWhenWeGetBackInfoThenItShouldReturnListWithElem
         assertEquals(4L, cloudEncryptionKeys.getCloudEncryptionKeys().size());
     }
 
+    @Test
+    public void collectEncryptionKeysWhenWeGetBackInfoThenItShouldReturnListWithPaginatedElements() {
+        ListKeysResult listKeysResult = new ListKeysResult();
+
+        Set<KeyListEntry> listEntries = new HashSet<>();
+        for (int i = 0; i < 52; i++) {
+            listEntries.add(keyListEntry(i));
+        }
+
+        listKeysResult.setKeys(listEntries);
+
+        DescribeKeyResult describeKeyResult = new DescribeKeyResult();
+        describeKeyResult.setKeyMetadata(new KeyMetadata());
+
+        ListAliasesResult describeAliasResult = new ListAliasesResult();
+
+        Set<AliasListEntry> aliasListEntries = new HashSet<>();
+        for (int i = 0; i < 52; i++) {
```

Comment:
Same here. If possible, I would like to see a parametrized loop with the argument as an input.